subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -1633,124 +1633,159 @@ int nrf_cloud_format_single_cell_pos_req_json(cJSON *const req_obj_out)
 	return err;
 }
 
-int nrf_cloud_format_cell_pos_req_json(struct lte_lc_cells_info const *const inf,
```

Comment:
Github absolutely butchered this diff

Reply:
These functions LGTM, though. I'm not familiar enough with the endpoints or cell info to comment on whether what you're adding or how makes sense, but structurally I follow the program and it looks clean

Reply:
yeah its really hard to read this diff.
just looking at the plain file will be more obvious.